traffic_ops/app/lib/API/DeliveryServiceRegexes.pm

```diff
@@ -27,12 +28,18 @@ use Validate::Tiny ':all';
 sub all {
 	my $self = shift;
 
+	my $tenant_utils = Utils::Tenant->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+
 	my $rs;
-	if ( &is_privileged($self) ) {
+	if ( &is_privileged($self) or $tenant_utils->ignore_ds_users_table()) {
```

Comment:
I thought you were changing this to check the 'use-tenancy' parameter so i would expect something like this to be:

if ( &is_privileged($self) or $tenant_utils->use_tenancy()) {